css/style.css

```diff
@@ -763,3 +763,41 @@ html {
   margin : 10px;
   align-content: center;
 }
+body{
+	margin:0;
+	padding:0;
+	font-family:"arial",heletica,sans-serif;
+	font-size:12px;
+    background: #2980b9 url('humblefool/img/background.png') repeat 0 0;
```

Comment:
url will be `../humblefool/img/background.png` as humblefool directory is not in the same as the one your file currently iis